routes/courses.js

```diff
@@ -61,8 +47,14 @@ router.get('/', async (req, res) => {
 
     query += ' ORDER BY start_year DESC';
 
-    let filtered = await client.query(query);
-    filtered = filtered.rows;
+    // TODO: running twice when the courses page gets loaded
```

Comment:
This comment is bit confusing. Is this mean that in the future this code is suppose to run twice when the page is loading? or is some odd behavior that needs to be looked into? Also, was this confirmed that this query is being ran twice within a single http request? Or is frontend making `two` requests?